src/main/kotlin/com/mparticle/kits/RoktKit.kt

```diff
@@ -185,19 +178,30 @@ class RoktKit : KitIntegration(), CommerceListener, IdentityListener, RoktListen
     }
 
     override fun onLoad() : Unit{
-        onLoadCallback?.run()
+        mpRoktEventCallback?.onLoad()
     }
 
     override fun onShouldHideLoadingIndicator() : Unit {
-        onShouldHideLoadingIndicatorCallback?.run()
+        mpRoktEventCallback?.onShouldHideLoadingIndicator()
     }
 
     override fun onShouldShowLoadingIndicator() : Unit {
-        onShouldShowLoadingIndicatorCallback?.run()
+        mpRoktEventCallback?.onShouldShowLoadingIndicator()
     }
 
     override fun onUnload(reason: Rokt.UnloadReasons) : Unit {
-        onUnloadCallback?.run()
+        mpRoktEventCallback?.onUnload(
+            when (reason) {
+                Rokt.UnloadReasons.NO_OFFERS -> MParticle.UnloadReasons.NO_OFFERS
+                Rokt.UnloadReasons.FINISHED -> MParticle.UnloadReasons.FINISHED
+                Rokt.UnloadReasons.TIMEOUT -> MParticle.UnloadReasons.TIMEOUT
+                Rokt.UnloadReasons.NETWORK_ERROR -> MParticle.UnloadReasons.NETWORK_ERROR
+                Rokt.UnloadReasons.NO_WIDGET -> MParticle.UnloadReasons.NO_WIDGET
+                Rokt.UnloadReasons.INIT_FAILED -> MParticle.UnloadReasons.INIT_FAILED
+                Rokt.UnloadReasons.UNKNOWN_PLACEHOLDER -> MParticle.UnloadReasons.UNKNOWN_PLACEHOLDER
+                Rokt.UnloadReasons.UNKNOWN -> MParticle.UnloadReasons.UNKNOWN
+            }
```

Comment:
This looks like it's missing from the iOS callback PR.  Should we discuss?  Looks like this code was added to [android pr here](https://github.com/mParticle/mparticle-android-sdk/pull/569/files#diff-f4a1258b96bf2d32b0f0f18cf0c60670487073660e3a0503491453f24d3f4575R1812-R1868), and I don't see an [equivalent on iOS](https://github.com/mParticle/mparticle-apple-sdk/pull/348/files).